# Rule to delete unnecessary console.log (no-console-custom)

Please describe the origin of the rule here.


## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js

// fill me in

```

Examples of **correct** code for this rule:

```js

// fill me in

```

### Options

If there are any options, describe them here. Otherwise, delete this section.

## When Not To Use It

Give a short description of when it would be appropriate to turn off this rule.

## Further Reading

If there are other links that describe the issue this rule addresses, please include them here in a bulleted list.

## Casos que queremos remover console.log

Se o pai dele for um bloco e o console não é o único elemento do bloco

https://astexplorer.net/#/gist/ab12d412ad08ea68f8dde21558721264/c002e9ce5c7503de35a5ccfe2e5361674f79e185

## comparação nos ifs

https://astexplorer.net/#/gist/b23c0f35c0800ab902a241093b7892c4/ba769b6b587a79d0d6f35fa5c6d63d02a0bbfb36

## comparação com true or false

https://astexplorer.net/#/gist/60e8dc9cb4d46122c43b92f77f541896/fd8c8d5f7ad16bf66f5f6ea8ecd6a37619e993ce